srcpkgs/libportal/template

```diff
@@ -1,22 +1,24 @@
 # Template file for 'libportal'
 pkgname=libportal
-version=0.4
+version=0.6
```

Comment:
see #35225

Reply:
That PR probably won't be merged because it patches things that don't need to be patched if with v42. I just wanted you to look at it and pull in relevant changes.

Reply:
Oh. My bad, I'll re-add it when I am available to do so.

Reply:
Alright, I checked the following against the new libportal package
+ nautilus
+ gnome-contacts
+ eog
+ gcolor3
+ epiphany
+ gnome-todo
+ gnome-builder

Let me know if I missed something.